elb_automation/meaoelb/oregon_b.py

```diff
@@ -29,22 +28,24 @@
 # but elb_tool.define_elb() fills in most of the blanks for you
 
 # add an IdleTimeout as an ELB attribute
-bedrock_stage.elb_config.elb_atts = ELBAtts(ELBAttIdleTimeout(120))
+bedrock_stage.elb_config.elb_atts.connection_settings.idle_timeout = 120
 # custom health check configuration
 bedrock_stage.elb_config.health_check.target_path = '/healthz/'
 
 
-### Bedrock Dev
+# ### Bedrock Dev
 bedrock_dev = elb_tool.define_elb_http(
     service_namespace='bedrock-dev',
     service_name='bedrock-nodeport',
     ssl_arn='arn:aws:acm:us-west-2:236517346949:certificate/657b1ca0-8c09-4add-90a2-1243470a6b45')
-bedrock_dev.elb_config.elb_atts = ELBAtts(ELBAttIdleTimeout(120))
+bedrock_dev.elb_config.elb_atts.connection_settings.idle_timeout = 120
 bedrock_dev.elb_config.health_check.target_path = '/healthz/'
 
 
 # show the ELB's before we process them
-elb_tool.show_elbs()
+# object output is now colorized JSON
+# elb_tool.show_elbs()
```

Comment:
why is this commented out now? Don't you want your fancy colorized JSON anymore? ;)

Reply:
it's a lot of output, and mostly useful when defining an ELB. Once defined, it's nice to leave json output commented out.